### PR TITLE
Add notes about release reviewers.

### PR DESCRIPTION
**Pull request by alibotean**
Scope
=====

I felt that the support team (for the moment @adiroiban and me) need to be up to date with all the changes in the product.

Since I was only skimming the news/release emails and not diving up in the details (reading the PR descriptions, glancing over the code changes) I've told @adiroiban  that it might be a good idea to make all support people part of the reviewers list when a new release PR is created.

If you feel the tone/wording/etc. is not ok or might not even be needed please share your thoughts.

reviewers @adiroiban @brunogola @lgheorghiu

### CODE REVIEW

**Comment by alibotean** [comment]
needs-review

**Comment by adiroiban** [comment]
thanks. looks good.

I have also added info about having the news article as part of the release PR... so that the news article will not pass unreviewed :)

changes-approved

**Comment by lgheorghiu** [comment]
changes-approved

File: content/release.rst [hunk 1]
```diff
@@ -20,6 +20,20 @@ Releasing a product usually consists of publishing the following:
...
 
 
+Release Review Process
+======================
+
+As a means to allow all team members that are involved in
+coding, testing and supporting the product to keep up with the changes,
+they must be part of the reviewers of the release branch/PR.
+
+Being part of the release review will help to ackwnoledge the changes and to
```
Comment (brunogola) [inline content/release.rst]: ackwnoledge -> acknowledge

**Comment by brunogola** [comment]
small typo

changes-approved

**Pull request merged**